themes-default/slim/gulpfile.js

```diff
@@ -59,7 +59,8 @@ const webpackedJsFiles = [
     'static/js/index.js',
     'static/js/api.js',
     'static/js/router.js',
-    'static/js/store/**/*.js'
+    'static/js/store/**/*.js',
+    'static/js/templates/**/*'
```

Comment:
While you have this open. I'm thinking we move templates to components since we now store components in there and we don't really need that exposes as a static file handler anymore either then since we don't have `httpVueLoader`.

Reply:
Oh I see, move them to the root folder of the `slim` theme?

Reply:
I was thinking `static/js/components` for now but removing the templates static file handler so they don't get exposed anymore. Actually now that you removed the copy that's probably fixed that anyways.

Reply:
> I was thinking `static/js/components` for now

Yeah but theme will end up being a bundled project, so the `js` folder would only hold the webpack bundles, IMO

> but removing the templates static file handler

It's actually a static file handler for the `js` folder. I don't think there's another way to not serve the `templates` subfolder other than to split the handler, move the folder itself.

> Actually now that you removed the copy that's probably fixed that anyways.

Yes, there won't be any files to serve from that folder. Just need make sure older PRs don't reintroduce the files.